docs/EXTENDED-ARCHIVES.md

```diff
@@ -291,15 +299,14 @@ A markdown file including post-flash instructions.
 
 A plain text file describing the image version's release notes.
```

Comment:
Any good reason why the Instructions are markdown, but the ReleaseNotes are plain text? Maybe we should allow both markdown or plain text for both?